### PR TITLE
rename mpt2 to mpo in BJ's mathbox

### PR DESCRIPTION
**Pull request by jkingdon**
Similar to the other mpo renames.

### CODE REVIEW

**Review by tirix — APPROVED** [review]
Looks good, but let's wait for @benjub's approval.

**Review by benjub — APPROVED** [review]
Good for me, thanks !

**Review by avekens — APPROVED** [review]

**Pull request merged**
